Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputModule.cs

```diff
@@ -191,8 +191,22 @@ protected MouseState GetMousePointerEventDataForMrtkPointer(PointerData pointerD
 
             // Reset the RaycastCamera for projecting (used in calculating deltas)
             Debug.Assert(pointer.Rays != null && pointer.Rays.Length > 0);
-            RaycastCamera.transform.position = pointer.Rays[0].Origin;
-            RaycastCamera.transform.rotation = Quaternion.LookRotation(pointer.Rays[0].Direction);
+
+            if (pointer.Controller != null && pointer.Controller.IsRotationAvailable)
+            {
+                RaycastCamera.transform.position = pointer.Rays[0].Origin;
+                RaycastCamera.transform.rotation = Quaternion.LookRotation(pointer.Rays[0].Direction);
+            }
+            else
+            {
+                // The pointer.Controller does not provide rotation, for example on HoloLens 1 hands.
+                // In this case pointer.Rays[0].Origin will be the head position, but we want the 
+                // hand to do drag operations, not the head.
+                // pointer.Position gives the position of the hand, use that to compute drag deltas.
```

Comment:
Is this assuming the GGVPointer is being used?

Reply:
I don't think so. But it does assume that if a pointer does not provide rotation, then we should be using the pointer position instead of pointer.Rays[0].Origin. I couldn't think of any other way to support GGV hands without explicitly saying "if controller is a ggvhand" or "if input source is a ggvhand". I didn't want to do that check because then we would be introducing a dependency between .Services and .Providers, which I don't think exists right now. 

If you or anybody else has a suggestion for how to check "if we are on HoloLens 1 or with a GGV hand, use the position of the hand" I'm very interested! I myself am not happy with this, but couldn't think of a better way.